src/linter.ts

```diff
@@ -23,6 +23,7 @@ import {
 
 namespace CommandIds {
   export const runLint: string = "shellcheck.runLint";
+  export const addIgnoreComment: string = "shellcheck.addIgnoreComment";
```

Comment:
We should probably update this too

```suggestion
  export const addIgnoreComment: string = "shellcheck.disableCheckForLine";
```

Or `shellcheck.disableForLine`

And maybe also rename the symbols, like variables and functions?

Reply:
YES! I'll update the variables, etc.

Reply:
Completed.